src/LayerPlan.cpp

```diff
@@ -1044,16 +1092,21 @@ void LayerPlan::addSplitWall(
 
     for (size_t point_idx = 1; point_idx < max_index; point_idx++)
     {
-        const ExtrusionJunction& p1 = wall[(wall.size() + start_idx + point_idx * direction) % wall.size()];
+        const size_t actual_point_index = (wall.size() + start_idx + point_idx * direction) % wall.size();
+        const Point2LL& p1 = wall.pointAt(actual_point_index);
+        const coord_t w1 = wall.lineWidthAt(actual_point_index);
 
-        if (! bridge_wall_mask_.empty())
+        if constexpr (std::is_same_v<PathType, ExtrusionLine>)
```

Comment:
I guess it makes sense that a `Polygon` is never bridging -- and that's the only other `PathType` at the moment, and we definitely don't _plan_ to add more... otherwise I'd say that you want the inversion maybe, select by it _not_ being a `Polygon`. Or maybe add a `bool` field/function to `PathType` to say if it's bridging or not.

That may be overkill given the situation though ... just maybe add some documentation here?

Reply:
I also added the `compute_distance_to_bridge_start` argument to make it a bit more explicit, but in the case of a Polygon it will just be silently ignored which is definitely not great... The documentation of this settings mentions that it will only work with ExtrusionLine though. I'll add some documentation in the function itself.